library/kani/src/lib.rs

```diff
@@ -94,10 +99,36 @@ pub fn any<T: Arbitrary>() -> T {
 /// kani::assume(char::from_u32(c as u32).is_ok());
 /// ```
 ///
+#[inline(never)]
+pub unsafe fn any_raw<T>() -> T
+where
+    [(); std::mem::size_of::<T>()]:,
```

Comment:
maybe add a comment about what this `where` is doing so people don't have to figure out on their own.